pkgs/development/python-modules/pgvector/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, asyncpg
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, numpy
+, postgresql
+, postgresqlTestHook
+, psycopg
+, psycopg2
+, pytestCheckHook
+, pythonOlder
+, sqlalchemy
+}:
+
+buildPythonPackage rec {
+  pname = "pgvector";
+  version = "0.1.8";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "pgvector";
+    repo = "pgvector-python";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-+0daPPZGVkqfZw0gOZwIl+knL/zZki9fs5kA3dYqPpE=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+  ];
+
+  nativeCheckInputs = [
+    asyncpg
+    django
+    (postgresql.withPackages (p: with p; [ pgvector ]))
+    postgresqlTestHook
+    psycopg
+    psycopg2
+    pytestCheckHook
+    sqlalchemy
+  ];
+
+  env = {
+    PGDATABASE = "pgvector_python_test";
+    postgresqlEnableTCP = 1;
+    postgresqlTestUserOptions = "LOGIN SUPERUSER";
+  };
+
+  # tests rely on network sandbox which is not available on darwin
+  doCheck = stdenv.isLinux;
```

Comment:
Can this be fixed by setting ``__darwinAllowLocalNetworking = true;``?

Reply:
Thanks.
Actually, the test passed without this variable.
I set [postgresqlEnableTCP](https://nixos.org/manual/nixpkgs/stable/#sec-postgresqlTestHook-tcp) and thought it was not available in darwin.